### PR TITLE
Add new recipe for bcgtree

### PR DESCRIPTION
**Pull request by jamesabbott**
Add new recipe for bcgtree

### CODE REVIEW

**Comment by jamesabbott** [comment]
@bioconda-bot please add label

File: recipes/bcgtree/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - {{ pin_subpackage('bcgtree', max_pin="{{ version }}") }}
+    
+requirements:
+  host:
+    - perl
+  run:
+    - perl
+    - python
+    - bash
+    - prodigal =2.6.3
```
Comment (bgruening) [inline recipes/bcgtree/meta.yaml:26]: Are those strict versions necessary? We usually try to keep the version pinning as flexible as possible.

Comment (jamesabbott) [reply recipes/bcgtree/meta.yaml:26]: Possibly not - they are the versions required according to the docs but I haven't tried using older versions. I could make them minimum required versions, or just remove the pinnings and see what happens...

File: recipes/bcgtree/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+source:
+  url: https://github.com/molbiodiv/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('bcgtree', max_pin="{{ version }}") }}
```
Comment (martin-g) [inline recipes/bcgtree/meta.yaml:17]:
```suggestion
    - {{ pin_subpackage('bcgtree', max_pin="x") }}
```

Comment (martin-g) [reply recipes/bcgtree/meta.yaml:17]: See https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md#instructions-for-avoiding-api-abi-and-cli-breakage-issues

File: recipes/bcgtree/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - raxml 
+
+test:
+  commands:
+    - bcgTree.pl --version
+
+about:
+  home: https://github.com/molbiodiv/{{ name }}
+  summary: Automatized phylogenetic tree building from bacterial core genomes
+  license: GPL
```
Comment (bgruening) [inline recipes/bcgtree/meta.yaml:39]: Can you please turn this into an SPDX identifier?

**Review by bgruening — APPROVED** [review]

**Pull request merged**